winml/lib/Api.Ort/OnnxruntimeEnvironment.cpp

```diff
@@ -13,10 +13,31 @@ using namespace _winml;
 
 static bool debug_output_ = false;
 
+
+static bool IsCurrentModuleInSystem32() {
```

Comment:
this appears to be a direct copy of the other one. do we have a folder where you can share it somehow?

Reply:
I had kept the adapter and winml codebases forked to avoid issues where one layer may move and another refactoring effort will need to take place.

Reply:
@martinb35 Where is the other copy?

Reply:
I also prefer it if the code was able to be shared but I'm not going to block the PR for this reason.

Reply:
keeping them forked.